environment-3.7.yml

```diff
@@ -37,7 +37,7 @@ dependencies:
 - plotly=4.0.0
 - BeautifulSoup4
 - make
-#- setuptools=41.5.1 # TODO: remove this when django 3 is used
 - pip:
     - django-leaflet==0.26.0
     - django-maintenance-mode
+    - ibis_framework==1.3.0
```

Comment:
it is good to add an empty line at the end of the file. this red icon here means that there is no empty line at the end of the file. check if your IDE is configured to add this extra line.

Reply:
ok!

Reply:
> it is good to add an empty line at the end of the file. this red icon here means that there is no empty line at the end of the file. check if your IDE is configured to add this extra line.

ok!

Reply:
Thanks for your review @xmnlab!